popupwindow.js

```diff
@@ -0,0 +1,111 @@
+const closeButton = document.getElementById('closeButton');
+const body1 = document.querySelector('body');
+
+closeButton.addEventListener('click', (e) => {
+  e.stopPropagation();
+  document.querySelector('#inPop').style.display = 'none';
+  document.querySelector('#popup').style.display = 'none';
+  document.getElementById('popup-content').style.display = 'none';
+  body1.style.overflow = 'scroll';
+});
+
+window.addEventListener('load', (event) => {
+  event.stopPropagation();
+  document.querySelector('#popup').style.display = 'none';
+  document.querySelector('#inPop').style.display = 'none';
+  document.getElementById('popup-content').style.display = 'none';
+});
+
+const projectDetails = [{
```

Comment:
- [ ] Kindly make sure to use this array to generates the work section dynamically as requested in the project description :recycle: 

For this you can proceed as follow:

```
const workSectipn= document.getElementById('work-section')

projectDetails.ForEach(project=> workSection.innerHtml+=`<div>
<h2>{project.title}</h2>
<img src={project.image}>
.....
`)
```